### PR TITLE
added registryLogLevel, can be set by user

### PR DESCRIPTION
**Pull request by vikrakum**
issue #119 
now user will be able to set both LOG_LEVEL and REGISTRY_LOG_LEVEL separately.

### CODE REVIEW

**Comment by andreaTP** [comment]
Hi @vikrakum sorry for the delay and thanks for the PR!
The code looks good, can you add an integration test for this feature?

**Comment by vikrakum** [comment]
Sure, i will.

File: api/v1/apicurioregistry_types.go [hunk 1]
```diff
@@ -81,10 +82,8 @@ type ApicurioRegistrySpecConfigurationSecurity struct {
 }
 
 type ApicurioRegistrySpecConfigurationSecurityHttps struct {
-	Enabled     bool   `json:"enabled,omitempty"`
+	DisableHttp bool   `json:"disableHttp,omitempty"`
```
Comment (andreaTP) [inline api/v1/apicurioregistry_types.go:85]: this is going to result in an API breaking change, correct @jsenko ?

**Review by andreaTP — CHANGES_REQUESTED** [review]

File: api/v1/apicurioregistry_types.go (hunk 1 above)
Comment (jsenko) [reply api/v1/apicurioregistry_types.go:85]: No because this feature has not been present in a previous release.

Comment (andreaTP) [reply api/v1/apicurioregistry_types.go:85]: Oh, wow! thanks! Where can I find the last released version of this file?

**Comment by jsenko** [comment]
hi @vikrakum can you please rebase your changes on top of main? So we avoid the merge commit.

**Comment by jsenko** [comment]
Thank you Vikram, great job!

**Comment by pantaoran** [comment]
Was this released yet? I just tried to set this, but it sent the operator into a crash-loop... (version 2.4.4 from RedHat)
- name: REGISTRY_LOG_LEVEL
   value: debug

**Pull request merged**